src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/PowerShellStepTest.java

```diff
@@ -0,0 +1,45 @@
+package org.jenkinsci.plugins.workflow.steps.durable_task;
+
+/*
+ * The MIT License
+ *
+ * Copyright 2017 Gabriel Loewen
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+import org.junit.Rule;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import hudson.model.Result;
+
+public class PowerShellStepTest {
+
+    @Rule public JenkinsRule j = new JenkinsRule();
+
+    @Test public void smokeTest() throws Exception {
```

Comment:
Needs to be guarded by assumptions similar to those in the upstream PR, or it will fail on Linux etc. (as it did in the CI job here).